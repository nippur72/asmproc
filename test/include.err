TEXT
include "include_1.lm"
TEXT
