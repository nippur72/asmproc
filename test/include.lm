TEXT
include "test/include_1.lm"
TEXT