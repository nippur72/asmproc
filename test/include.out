TEXT
FILE INCLUDE_1.LM
FILE INCLUDE_2.LM
HELLO!
FILE INCLUDE_2.LM
FILE INCLUDE_1.LM
TEXT